src/rust/src/conversion/mod.rs

```diff
@@ -862,3 +862,17 @@ pub(crate) fn parse_parquet_compression(
     };
     Ok(parsed)
 }
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
Is this needed?

Reply:
No, I forgot that wasm supports sinks, just not parquet